ui/pages/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -98,6 +98,10 @@ export default class DomainInput extends Component {
     const { className, selectedAddress, selectedName, userInput } = this.props;
 
     const hasSelectedAddress = Boolean(selectedAddress);
+    let isFlask = false;
```

Comment:
```suggestion
    const isFlask = process.env.METAMASK_BUILD_TYPE === 'flask';
```
Wondering if this should be a runtime check without fencing? This makes testing easier too.